config/channels.go

```diff
@@ -48,6 +53,62 @@ func LoadChannelsFromPl(location string) error {
 	return errors.New("No streams in playlist")
 }
 
+func WatchChannelsFile() {
+    watcher, err := fsnotify.NewWatcher()
+    if err != nil {
+        log.Fatal(err)
+    }
+    defer watcher.Close()
+
+    err = watcher.Add("channels.json")
+    if err != nil {
+        log.Fatal(err)
+    }
+
+    for {
+        select {
+        case event, ok := <-watcher.Events:
+            if !ok {
+                return
+            }
+            if event.Op&fsnotify.Write == fsnotify.Write {
```

Comment:
minor nit: would this work as well?

```suggestion
            if event.Has(fsnotify.Write) {
```

I'm not familiar with fsnotify, but I noticed this difference from the example on their README.